.github/workflows/ci.yaml

```diff
@@ -457,7 +457,7 @@ jobs:
       - name: Compute version number
         run: |
           VERSION=$(printf 'VER\t${project.version}' | mvn help:evaluate | grep '^VER' | cut -f2)
-          if [ "${GITHUB_REF}" = "refs/heads/master" ]
+          if [ "${GITHUB_REF}" = "refs/heads/arm64-images" ]
```

Comment:
question:
Shouldn't this be reverted to `master` once this PR is merged?

Reply:
good catch, let me change it before merging